aws/resource_aws_rds_cluster_instance.go

```diff
@@ -296,6 +301,7 @@ func resourceAwsRDSClusterInstanceRead(d *schema.ResourceData, meta interface{})
 	d.Set("promotion_tier", db.PromotionTier)
 	d.Set("preferred_backup_window", db.PreferredBackupWindow)
 	d.Set("preferred_maintenance_window", db.PreferredMaintenanceWindow)
+	d.Set("availability_zone", db.AvailabilityZone)
```

Comment:
Could you augment an existing acceptance test to check for this to be set? perhaps the basic one?